File: programmers/python 3/12949.py
# 문제 링크: https://programmers.co.kr/learn/courses/30/lessons/12949

def solution(arr1, arr2):
    answer = [[0] * len(arr2[0]) for _ in range(len(arr1))]
    for i in range(len(arr1)):
        for j in range(len(arr2[0])):
            answer[i][j] = sum(
                arr1[i][k] * arr2[k][j]
                for k in range(len(arr2))
            )
    return answer

File: programmers/swift/12949.swift
// 문제 링크: https://programmers.co.kr/learn/courses/30/lessons/12949

func solution(_ arr1: [[Int]], _ arr2: [[Int]]) -> [[Int]] {
    arr1.indices.map { i in
        arr2[0].indices.map { j in
            arr2.indices
                .map { k in arr1[i][k] * arr2[k][j] }
                .reduce(0, +)
        }
    }
}